superset-frontend/src/dashboard/util/permissionUtils.test.ts

```diff
@@ -22,7 +22,11 @@ import {
 } from 'src/types/bootstrapTypes';
 import { Dashboard } from 'src/types/Dashboard';
 import Owner from 'src/types/Owner';
-import { canUserEditDashboard, isUserAdmin } from './permissionUtils';
+import {
+  canUserAccessSqlLab,
+  canUserEditDashboard,
+  isUserAdmin,
+} from './permissionUtils';
```

Comment:
TODO: these should probably be moved to a general util outside the dashboard section, but refactoring it seems outside the scope of this PR